modules/ROOT/pages/update-modules.adoc

```diff
@@ -21,4 +21,20 @@ image::update-all-modules.png[]
 
 Studio updates your selected dependencies.
 
-Warning: If you are using Maven properties to set the connector version, then Studio will show the availability for the update, but will not perform the update. You will need to contact your Development team and change your custom maven code to update to the last version available
+This method does not work if you use Maven properties to specify your modules' version:
```

Comment:
I would try to avoid the negative here, but I don't have a good alternative to offer. 

"This method works only if you do not use..." etc. Maybe. But this could be more confusing if they only glance at it. Up to you.

Reply:
Maybe avoid the negative altogether? 

>Note that if you use Maven properties to specify your modules version: